plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -45,6 +45,8 @@ struct ffmpeg_muxer {
 	int64_t stop_ts;
 	uint64_t total_bytes;
 	struct dstr path;
+	/* printable_path is path with any stream key information removed */
+	const char *printable_path;
```

Comment:
Use struct dstr for printable_path, then you don't need to worry about if the array pointed by the pointer could be deallocated or not.